Assets/BossRoom/Prefabs/UI/PartyHUD.prefab

```diff
@@ -515,12 +515,12 @@ MonoBehaviour:
   m_HeroPortrait: {fileID: 5210474332450112926}
```

Comment:
At a glance, this seems like same change as the one done in PR 281. Does this change need to come through this PR?

Reply:
yep, this diff has gone away, as you can see.

Reply:
it was needed for working on the PR, but the commit has now been taken into develop, and doesn't show up as a diff anymore.